### PR TITLE
Microverse React capstone project

### PR DESCRIPTION
**Pull request by Fombi-Favour**
In this activity, we: 
- Setup the environment + linters
- Added `Redux`, `Redux-toolkit`, `React-Router` and other packages
- Setup routing functions
- Added Redux store and and corresponding slices
- Created home page displaying continents
- Created detail page displaying countries of the continents
- Implement unit testing

## 🎥 Presentation
- Check out the presentation for this project [Click here!](https://www.loom.com/share/d66523b5ec07413c8b23c4582bd99e7f?sid=5147dce9-b707-4f57-8c9c-70db11e177b4)

## Warning ⚠️ 
- Make sure your internet is stable to view every information (mostly the details page).
- Every time you visit the details page the second time, make sure you refresh the page to view updates.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,182 @@
...
+2.- To see the code open the project in the code editor of your preference.
+
+## Warning ⚠️ 
+- Make sure your internet is stable to view every information (mostly the details page).
+- Every time you visit the details page the second time, make sure you refresh the page to view updates.
+
+### Deployment
+
+You can deploy this project on github following instructions here:
+[deploy website on github](https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site)
```
Comment (Tufahel) [inline README.md:133]: - [ ] Kindly add the `Live Demo Version` of your website using `Netlify`, `Render` or `Github Pages` and add this link here or `Live Demo Version` section.

File: src/components/Country.jsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+            deaths
+          </span>
+          <span>
+            Recovered:
+            {' '}
+            {recovered}
+            {' '}
+            recovered
+          </span>
+        </div>
```
Comment (Tufahel) [inline src/components/Country.jsx:44]: - First when I click on a specific `Continent` then it goes to the `Continet Page` and shows the countries below. It is fine.
- But, there is an issue. As per the given design guideline, there should be a page for country details, When I click on each country it doesn't open any page with `Country details`. Country details shouldn't be shown on the `Continent` page, country details should be shown on a different page.
- Suppose, I click on the `Continent Asia`, then the `Asia` continent page opens. Then I click on a `Country` named `Bangladesh`, it should open a page with the details `covid` report of `Bangladesh` in a new page.
- [ ] Kindly fix this issue.

- ***Given Design Guideline***
image

**Review by Tufahel — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @Fombi-Favour :wave:  

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🌟

Alt Text

✔️ Descriptive pull request.
✔️ All tests are passing.
✔️ Presentation is professional.
✔️ Implemented filter for searching.

## Required Changes ♻️

***Please kindly check the comments/review below.***

## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.
- You can mail me at [Mail](mailto:tufahel96@gmail.com) or ask any question at Slack, `Name: Tufahel`

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ghost — APPROVED** [review]
Hi @Fombi-Favour , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 





## Highlights

✅ Correct use of Gitflow
✅ No linter error
✅ Good work testing your application properly 👍

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**